tests-scan

```diff
@@ -181,28 +181,20 @@ def queue_test(job: Job, channel, options: argparse.Namespace) -> None:
         print(json.dumps(body['job']))
 
 
+def update_status(
+    api: github.GitHub, revision: str, last: Mapping[str, str], dry: bool, changes: Mapping[str, str] | None
+) -> bool:
+    if changes is None or {**last, **changes} == last or dry:
+        return True
```

Comment:
The commit message claims "use task.would() to print a nice message in that case", but it doesn't?

Reply:
It did but it broke the expected output of the tests so I took it back out.  I'll fix the commit message.